aws-auth-plugins-core/src/main/java/com/amplifyframework/auth/plugins/core/data/AWSCognitoIdentityPoolConfiguration.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.auth.plugins.core.data
+
+/**
+ * Configuration options for specifying cognito identity pool.
+ */
+data class AWSCognitoIdentityPoolConfiguration(val poolId: String, val region: String = "us-east-1")
```

Comment:
Why hardcode this?

Reply:
Region is optional. This is in line with SDK which also defaults to `us-east-1`.